cmd/statusd/library.go

```diff
@@ -386,6 +386,25 @@ func makeJSONResponse(err error) *C.char {
 
 //export Notify
 func Notify(token *C.char) *C.char {
-	res := statusAPI.Notify(C.GoString(token))
-	return C.CString(res)
+	err := statusAPI.Notify(C.GoString(token))
+
+	res := true
+	errString := ""
+	if err != nil {
+		res = false
+		errString = err.Error()
+	}
+
+	out := common.NotifyResult{
+		Status: res,
+		Error:  errString,
+	}
+
+	outBytes, err := json.Marshal(&out)
```

Comment:
Why a reference to `out` here?

Reply:
Hmm... i think it's copy-paste from this file.

Reply:
Let's fix it?

Reply:
Ok, done.